### PR TITLE
[TASK] "Custom Backend Module" documentation

### PR DESCRIPTION
**Pull request by mficzel**

### CODE REVIEW

**Comment by kitsunet** [comment]
Hey was almost merging this but then realized that the indentation of the code examples is not exactly correct. Would you go in there and fix it please? Then it's a merge for me.

**Comment by aertmann** [comment]
Should be targeted at 2.0 instead of master

File: TYPO3.Neos/Documentation/ExtendingNeos/CustomBackendModules.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
+.. _custom-backend-modules:
+
+Custom Backend Modules
+======================
+
+If you want to integrate custom tools into the neos-backend you can do so by adding a submodule to the
+administration- or the management-section of the main menu.
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/ExtendingNeos/CustomBackendModules.rst]: it's also possible to create one outside of those two..

File: TYPO3.Neos/Documentation/ExtendingNeos/CustomBackendModules.rst [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+Some possible use cases would be the integrating of mass editing options, triggering of import or export
+actions or creating of editing interfaces for DomainModels from FlowPackages.
+
+.. warning:: This is no pubic api yet and still might change in future releases of Neos.
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/ExtendingNeos/CustomBackendModules.rst]: This is not public API yet due to it's unpolished state and is subjectable to change in the future

**Comment by bwaidelich** [comment]
@mficzel very nice addition, thanks! Would you be up for re-applying this to the correct branch (2.0)?

**Comment by aertmann** [comment]
Cherry-picked this one and pushed a new commit on top with fixing of the mentioned issues and some more adjustments

See 74de538fc372a6e20035d83d33af77e088d59cc3 da6354b77ac3a306f03be9e2016c126380f3ed96

**Pull request closed without merging**